includes/common.php

```diff
@@ -1510,14 +1510,21 @@ function clean($value)
  * @param $value
  * @return string
  */
-function display($value)
+function display($value, $purifier_config = array())
 {
     /** @var HTMLPurifier $purifier */
     global $config, $purifier;
+
+    if (empty($purifier_config)) {
```

Comment:
Can you add a comment here, I can't figure out why this is here by looking at the code alone.